### PR TITLE
Migrate stack files to cpg-infra-private

### PR DESCRIPTION
**Pull request by illusional**
https://github.com/populationgenomics/cpg-infrastructure-private/pull/8

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,4 +3,5 @@ pulumi-gcp==6.49.0
 pulumi-azure-native==1.80.0
 pulumi-azuread==5.28.0
 toml==0.10.2
+xxhash==3.2.0
```
Comment (lgruen) [inline requirements.txt:6]: Was this intentional?

File: cpg_infra/abstraction/azure.py [hunk 2]
```diff
@@ -48,9 +48,9 @@ def __init__(
         self.storage_account_lifecycle_rules = []
         self.storage_account_undelete_rule = None
 
-        data = az.authorization.get_client_config()
-        self.subscription_id = '/subscriptions/' + data.subscription_id
-        self.tenant = data.tenant_id
+        # data = az.authorization.get_client_config()
```
Comment (lgruen) [inline cpg_infra/abstraction/azure.py:51]: This seems unrelated?

**Review by lgruen — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (illusional) [reply requirements.txt:6]: I included this and the below change as they made migrating the state easier.

This, so I can install directly with pip as it was missing. And the below change because it helped for Azure debugging, but then realised it wasn't all relevant anyway.

I can move these to a separate PR, but they do relate to the stack migration to infra private

File: cpg_infra/abstraction/azure.py (hunk 2 above)
Comment (illusional) [reply cpg_infra/abstraction/azure.py:51]: This was initially failing with some related az auth issue, but then it occurred through this migration process that it's not needed. I've removed the line instead of commenting it out.

**Review by lgruen — APPROVED** [review]
